strawberryfields/backends/base.py

```diff
@@ -703,41 +707,47 @@ def prepare_gaussian_state(self, r, V, modes):
         Args:
             r (array): vector of means in xp ordering
             V (array): covariance matrix in xp ordering
-            modes (int or Sequence[int]): which mode to prepare the state in
-                If the modes are not sorted, this is take into account when preparing the state.
-                i.e., when a two mode state is prepared in modes=[3,1], then the first
-                mode of state goes into mode 3 and the second mode goes into mode 1 of the simulator.
+            modes (int or Sequence[int]): Which modes to prepare the state in.
+                If the modes are not sorted, this is taken into account when preparing the state.
+                I.e., when a two mode state is prepared in modes=[3,1], the first
+                mode of the given state goes into mode 3 and the second mode goes into mode 1.
         """
         raise NotImplementedError
 
     def get_cutoff_dim(self):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
 
     def prepare_fock_state(self, n, mode):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
 
     def prepare_ket_state(self, state, mode):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
 
     def prepare_dm_state(self, state, mode):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
 
     def cubic_phase(self, gamma, mode):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
 
     def kerr_interaction(self, kappa, mode):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
 
     def cross_kerr_interaction(self, kappa, mode1, mode2):
-        # pylint: disable=unused-argument,missing-docstring
         raise NotApplicableError
 
     def measure_fock(self, modes, select=None):
-        # pylint: disable=unused-argument,missing-docstring
```

Comment:
Nice, pylint didn't used to be able to determine this automatically (must be due to keeping up with the new Sphinx functionality).